methods.py

```diff
@@ -1,33 +1,38 @@
-from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.wait import WebDriverWait
 
 
 class Methods(object):
-
+    """Webdriver wrapper"""
     def __init__(self, driver):
         self.driver = driver
 
-    def get_element(self, locator_type, locator, numb_list=0):
-        # This function seeks for elements
-        types_By = {
-            'id': By.ID,
-            'css': By.CSS_SELECTOR}
-        element = self.driver.find_elements(types_By[locator_type], locator)
-        return element[numb_list]
-
-    def click_element(self, locator_type, locator, numb_list=0):
-        element = self.get_element(locator_type, locator, numb_list)
-        element.click()
-
-
-    def clear_element(self, locator_type, locator):
-        element = self.get_element(locator_type, locator)
-        element.clear()
-
-    def send_keys(self, locator_type, locator, keys):
-        element = self.get_element(locator_type, locator)
-        element.send_keys(keys)
-
-
-            
+    def get_elements(self, locator):
+        """Returns all elements for the specific locator"""
+        return self.driver.find_elements(*locator)
+
+    def click_element(self, locator, elem_number=0):
```

Comment:
Lets drop this method in favor of `click_element_by_text`

Reply:
We can but later we will need it anyway

Reply:
Try to avoid redundant code now.

Reply:
ok